packages/config/config/devices/0x0234/zif5020_2.0.json

```diff
@@ -0,0 +1,208 @@
+		}
+	],
+	"firmwareVersion": {
+		"min": "2.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Report, Input 1",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Basic Switch Report, Input 1",
+			"maxNodes": 5
+		},
+		"4": {
+			"label": "Multilevel Switch Report, Input 1",
+			"maxNodes": 5
+		},
+		"5": {
+			"label": "Basic Report, Input 2",
+			"maxNodes": 5
+		},
+		"6": {
+			"label": "Basic Switch Report, Input 2",
+			"maxNodes": 5
+		},
+		"7": {
+			"label": "Multilevel Switch Report, Input 2",
+			"maxNodes": 5
+		},
+		"8": {
+			"label": "Basic Report, Input 3",
+			"maxNodes": 5
+		},
+		"9": {
+			"label": "Basic Switch Report, Input 3",
+			"maxNodes": 5
+		},
+		"10": {
+			"label": "Multilevel Switch Report, Input 3",
+			"maxNodes": 5
+		},
+		"11": {
+			"label": "Basic Report, Input 4",
+			"maxNodes": 5
+		},
+		"12": {
+			"label": "Basic Switch Report, Input 4",
+			"maxNodes": 5
+		},
+		"13": {
+			"label": "Multilevel Switch Report, Input 4",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "templates/logic_group_template.json#status_led"
+		},
+		"2": {
+			"$import": "templates/logic_group_template.json#led_brightness"
+		},
+		"3": {
+			"$import": "templates/logic_group_template.json#input_function_setup",
+			"label": "Function Setup Of Input 1"
+		},
+		"4": {
+			"$import": "templates/logic_group_template.json#input_timer",
+			"label": "Timer For Input 1"
+		},
+		"5": {
+			"$import": "templates/logic_group_template.json#input_function_setup",
+			"label": "Function Setup Of Input 2"
+		},
+		"6": {
+			"$import": "templates/logic_group_template.json#input_timer",
+			"label": "Timer For Input 2"
+		},
+		"7": {
+			"$import": "templates/logic_group_template.json#input_function_setup",
+			"label": "Function Setup Of Input 3"
+		},
+		"8": {
+			"$import": "templates/logic_group_template.json#input_timer",
+			"label": "Timer For Input 3"
+		},
+		"9": {
+			"$import": "templates/logic_group_template.json#input_function_setup",
+			"label": "Function Setup Of Input 4"
+		},
+		"10": {
+			"$import": "templates/logic_group_template.json#input_timer",
+			"label": "Timer For Input 4"
+		},
+		"11": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "Input Snubber-filter Time Constant.",
+			"unit": "50 ms",
+			"defaultValue": 5
+		},
+		"12": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "Threshold Value For Activation Of Inputs.",
+			"unit": "200 ms",
+			"defaultValue": 20
+		},
+		"13": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "Threshold For Input In Latched Mode.",
+			"unit": "50 ms",
+			"defaultValue": 50
+		},
+		"14": {
+			"$import": "templates/logic_group_template.json#enable_central_scene",
+			"defaultValue": 0,
+			"options": [
```

Comment:
You can't just invert the Enable/Disable like this becasue eventually these will be represented with a switch. Use the base_enable_disable_inverted base group from the master template instead.